### PR TITLE
Polish filter panel

### LINKED ISSUE #122: Use blue dot to indicate active filters instead of filter funnel icon

### PR DESCRIPTION
**Pull request by iangmhill**
Resolves #122 and #123.

Blue dots are now consistently used indicate active filters, and abbreviations in the filter panel have been expanded to avoid confusion.

### CODE REVIEW

File: ember/app/styles/components/discrete-filter-subpanel.scss [hunk 1]
```diff
@@ -1,41 +1,32 @@
 .component.subpanel.discrete-filter {
-
+  $discrete_filter_font_size: 14px;
```
Comment (ericyoungberg) [inline ember/app/styles/components/discrete-filter-subpanel.scss]: This doesn't look like it's being used anywhere else but one place

**Comment by ericyoungberg** [comment]
First item in each list that implements the blue dot, the blue dot is not vertically centered.

capture d ecran 2018-06-27 a 9 34 28 am

**Comment by iangmhill** [comment]
@ericyoungberg Changes made.

**Comment by iangmhill** [comment]
@ericyoungberg I consolidated the font-size declarations, but I cannot reproduce the development form bleeding.

**Comment by ericyoungberg** [comment]
I can't either. Must've been some freaky branch-switching freakout.

**Pull request merged**